mtp_cashbook/settings/base.py

```diff
@@ -44,7 +44,7 @@
 )
 
 PROJECT_APPS = (
-    'core',
+    'moj_utils',
```

Comment:
Not sure where that is, I get ImportError: No module named 'moj_utils' when running the app